README.md

```diff
@@ -2,55 +2,48 @@
 
 # Project Name
```

Comment:
Please add a project name in README. A professional README file should have all the details of a project.

Reply:
okay, thank you